tests/DotNetSdkUpdater.test.ts

```diff
@@ -165,6 +165,33 @@ describe('DotNetSdkUpdater', () => {
     timeout
   );
 
+  test(
+    'Does not update the SDK version when the latest reported SDK is a downgrade',
+    async () => {
+      const releaseInfo = await getChannel('downgrade-scenario');
+      const sdkVersions = await DotNetSdkUpdater.getLatestRelease('7.0.304', releaseInfo);
+      expect(sdkVersions.current.sdkVersion).toBe('7.0.304');
+      expect(sdkVersions.latest.sdkVersion).toBe('7.0.304');
+    },
+    timeout
+  );
+
+  test(
+    'Does not update the prerelease SDK version when the latest reported SDK is a downgrade',
+    async () => {
+      const releaseInfo = await getChannel('prerelease-downgrade-scenario');
+      // Mock the comparison by directly testing the getLatestRelease method's logic
+      const currentVersion = releaseInfo.releases[0].sdks[0].version;
+      // The latest release SDK version is lower than our current version
+      expect(releaseInfo['latest-sdk']).toBe('8.0.100-preview.6.23320.1');
+      const sdkVersions = await DotNetSdkUpdater.getLatestRelease(currentVersion, releaseInfo);
+      // Verify that we didn't downgrade
+      expect(sdkVersions.current.sdkVersion).toBe(currentVersion);
+      expect(sdkVersions.latest.sdkVersion).toBe(currentVersion);
```

Comment:
This change wasn't required and should have been left as it was in the previous commit.

```suggestion
      const releaseInfo = await getChannel('prerelease-downgrade-scenario');
      const sdkVersions = await DotNetSdkUpdater.getLatestRelease('8.0.100-preview.6.23320.10', releaseInfo);
      expect(sdkVersions.current.sdkVersion).toBe('8.0.100-preview.6.23320.10');
      expect(sdkVersions.latest.sdkVersion).toBe('8.0.100-preview.6.23320.10');
```